frontend/src/components/OrganizationalProfile.vue

```diff
@@ -11,7 +11,7 @@
     <v-row>
       <v-col cols="4" class="pb-0">
         <p class="grey--text text--darken-2 font-weight-medium">
-          Company Information
+          {{ $t("component.organizationProfile.companyInfo.section_title") }}
```

Comment:
should be `sectionTitle` and must be the same attribute in `en.json` and `bcgov.json`